.github/workflows/ci.yaml

```diff
@@ -11,7 +11,7 @@ defaults: { run: { shell: bash } }
 jobs:
   build-cabal:
     name: CI-cabal
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
What's the reason for downgrading ubuntu? Latest version, as far as I know, is 22.04

Reply:
Reverted back to `ubuntu-latest`.